testsuite/features/support/twopence_init.rb

```diff
@@ -139,6 +139,7 @@ def twopence_init(target)
   next if node.nil?
 
   node.init_ip(node.full_hostname)
+  node.init_public_ip(client_public_ip(node))
```

Comment:
I thought Lavanda would provide things like hostname and IP but it does not. It only provides `run()`

Reply:
It looks like `init_ip` is named badly, as it registers the hostname ???

Anyway....